time_manager/Makefile.am

```diff
@@ -23,7 +23,11 @@
 # Ed Hartnett 2/22/19
 
 # Include .h and .mod files.
-AM_CPPFLAGS = -I${top_builddir}/.mod -I${top_srcdir}/include
+AM_CPPFLAGS = -I${top_srcdir}/include
+AM_CPPFLAGS += -I${top_builddir}/platform
+AM_CPPFLAGS += -I${top_builddir}/constants
+AM_CPPFLAGS += -I${top_builddir}/fms
+AM_CPPFLAGS += -I${top_builddir}/mpp
```

Comment:
This seems to not be using the .mod/ directory where f90 modulefiles should be.

Reply:
@wrongkindofdoctor, .mod is also not used in drifters/Makefile.am
With these two Makefile.am's fixed, your branch compiled for me with Intel on Skylake :)